src/main/java/org/mondemand/Client.java

```diff
@@ -316,6 +315,14 @@ public void setErrorHandler(ErrorHandler errorHandler) {
     }
   }
 
+  /**
+   * @param maxNumMetrics the maximum number of metrics to send in stats messages
+   */
+  public void setMaxNumMetrics(Integer maxNumMetrics)
```

Comment:
I also left this it's own setter, as I don't want to make even more of a mess of the constructors than this class already has.